pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -359,17 +359,38 @@ func TestPriorityQueue_Delete(t *testing.T) {
 }
 
 func TestPriorityQueue_MoveAllToActiveOrBackoffQueue(t *testing.T) {
-	q := NewPriorityQueue(newDefaultQueueSort())
+	c := clock.NewFakeClock(time.Now())
+	q := NewPriorityQueue(newDefaultQueueSort(), WithClock(c))
 	q.Add(&medPriorityPod)
+	q.MoveAllToActiveOrBackoffQueue("test")
```

Comment:
why is this line necessary?

Reply:
I guess that is for updating the `PriorityQueue.moveRequestCycle`, maybe we can assert  the `PriorityQueue.moveRequestCycle` and `PriorityQueue.schedulingCycle` after this line, that help the people understand the purpose for this line.

Reply:
I pulled this line ahead compared with original testcase, combined with the line below `c.Step(time.Second)` just to make sure the move request is happened first, and `p.moveRequestCycle` got updated.

Reply:
> why is this line necessary?

not necessary actually, have reverted it back.